src/Softeq.NetKit.Chat.SignalR/Sockets/ChannelSocketService.cs

```diff
@@ -173,7 +168,8 @@ public async Task<ChannelResponse> InviteMemberAsync(InviteMemberRequest request
                 await _pushNotificationService.SubscribeUserOnTagAsync(invitedMember.SaasUserId, PushNotificationsTagTemplates.GetChatChannelTag(channel.Id.ToString()));
             }
 
-            await _channelNotificationService.OnJoinChannel(invitedMember, channel);
+            await _channelNotificationService.OnJoinChannel(channel, invitedMember);
+            await SendSystemMessageAsync(request.SaasUserId, request.ChannelId, new MemberJoinedLocalizationVisitor(invitedMember), RecipientType.AllExceptCallerConnectionId);
```

Comment:
We need to add new event type instead of MemberJoinedLocalizationVisitor. According documentation it should be something like MemberInvitedLocalizationVisitor.

Reply:
skipping

Reply:
agreed that currently we don't need this event type